src/main/java/seedu/address/ui/VolunteerCard.java

```diff
@@ -58,6 +62,12 @@ public VolunteerCard(Volunteer volunteer, int displayedIndex) {
         address.setText(volunteer.getAddress().value);
         age.setText(volunteer.getAge().value);
         email.setText(volunteer.getEmail().value);
+        region.getChildren().add(
+                new Label(volunteer.getRegion().region.name())
+        );
+        volunteer.getMedicalTags().stream()
+                 .sorted(Comparator.comparing(tag -> tag.tagName))
```

Comment:
Is there a need to sort it? I'm thinking if needed, it might be better to sort by `BASIC`, `INTERMEDIATE` and `ADVANCED` instead.